### PR TITLE
Init ui for user

### PR DESCRIPTION
**Pull request by nguyenducanhst96**
image

### CODE REVIEW

**Comment by nguyenducanhst96** [comment]
https://dev.framgia.com/issues/184602

**Comment by nguyenducanhst96** [comment]
ready

File: app/assets/images/Chinese.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 33.56 19.57"><defs><style>.cls-1{fill:none;}.cls-2{clip-path:url(#clip-path);}.cls-3{fill:#cb1d23;}.cls-4{fill:#f7de29;}</style><clipPath id="clip-path"><rect class="cls-1" width="33.56" height="19.57"/></clipPath></defs><title>Ресурс 8</title><g id="Шар_2" data-name="Шар 2"><g id="_4" data-name="4"><g id="CHINA"><g class="cls-2"><rect class="cls-3" width="33.56" height="19.57"/><polygon class="cls-4" points="3.23 7.09 3.86 5.13 2.2 3.92 4.25 3.91 4.89 1.97 5.53 3.91 7.58 3.92 5.93 5.13 6.55 7.09 4.89 5.88 3.23 7.09"/><polygon class="cls-4" points="8.95 8.14 9.64 8.42 10.13 7.85 10.07 8.59 10.77 8.88 10.04 9.06 9.99 9.8 9.59 9.17 8.87 9.34 9.35 8.77 8.95 8.14"/><polygon class="cls-4" points="12.7 4.39 11.97 4.26 11.61 4.92 11.51 4.18 10.77 4.04 11.44 3.72 11.34 2.98 11.86 3.51 12.53 3.19 12.18 3.85 12.7 4.39"/><polygon class="cls-4" points="11.14 7.73 11.37 7.02 10.77 6.58 11.51 6.58 11.74 5.87 11.98 6.58 12.72 6.58 12.12 7.02 12.35 7.73 11.74 7.29 11.14 7.73"/><polygon class="cls-4" points="8.99 1.24 9.66 1.56 10.18 1.01 10.09 1.75 10.76 2.07 10.03 2.21 9.94 2.96 9.57 2.31 8.84 2.44 9.35 1.9 8.99 1.24"/></g></g></g></g></svg>
```
Comment (dangtuan1996pro) [inline app/assets/images/Chinese.svg]: xuoongs dong

File: vendor/assets/javascripts/user/blog.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+		menuActive = true;
+	}
+
+	function closeMenu()
+	{
+		menu.removeClass('active');
+		menuActive = false;
+	}
+
+});
```
Comment (dangtuan1996pro) [inline vendor/assets/javascripts/user/blog.js]: xuống dòng

File: vendor/assets/javascripts/user/blog_single.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+	*/
+
+	function initSearch()
+	{
+		if($('.search_button').length)
+		{
+			var search = $('.search_button');
+			search.on('click', function()
+			{
+				$('.search_form_container').toggleClass('active');
```
Comment (dangtuan1996pro) [inline vendor/assets/javascripts/user/blog_single.js]: ident

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+              <a href="#">
+                <div class="logo_text"><%= t ".english" %></div>
+              </a>
+            </div>
+            <nav class="main_nav_contaner">
+              <ul class="main_nav">
+                <li class="active"><%= link_to t(".home"), static_pages_index_path %></li>
+                <li class="nav-item dropdown">
+                  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    Exam
```
Comment (dangtuan1996pro) [inline app/views/layouts/_header.html.erb]: i18n

Comment (nguyenducanhst96) [reply app/views/layouts/_header.html.erb]: ok

**Comment by nguyenducanhst96** [comment]
ready

**Pull request merged**